src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+
+public class Task {
+    private String title;
+    private Boolean isDone;
+
+    public Task(String title) {
+        this.title = title;
+        this.isDone = false;
+    }
+
+    /**
+     * Mark the task as done.
```

Comment:
```suggestion
     * Marks the task as done.
```
According to the coding guideline, should be marks, not mark :)